02_getting_started.asciidoc

```diff
@@ -9,6 +9,24 @@ The Lightning Network is accessed via software applications that can speak the L
 
 Users have the highest degree of control by running their own Bitcoin node and Lightning node. However, Lightning nodes can also use a lightweight Bitcoin client (commonly referred to as Simplified Payment Verification (SPV)) to partially validate the correctness of their blockchain.
 
+=== Lightning Explorer
+
+A Lightning network explorer is an useful tool to show the statistics of nodes, channels and network capacity.
+
+The below is an unexhausted list:
+
+* https://1ml.com/, 1ML Lightning explorer
+* https://lightblock.me/, Lightblock Lightning explorer
+* https://explorer.acinq.co/, ACINQ's Lightning explorer, with fancy visualization
+* https://www.robtex.com/lightning/node/, Robtex Lightning explorer
+
+However, using Lightning explorer **may introduce privacy issue** because, if users are careless, the website may track their IP addresses and collect their behavior records (for example, the nodes users are interested in).
```

Comment:
IMO the privacy implications aren't the most important aspect users should be notified about (not really much different from the existing set of block explorers). What we need to outline is that they're showing subjective information, as there's no global consensus of the current LN graph, nor the current state of any existing channel policies. As a result, they're not to be relied upon for the most up to date information, and should be used to glance at loosely consistent statistics concerning the network.

Reply:
Well I brought up the privacy implications and just because they are as bad as block explorers does not mean that we should ignore them. However i would also agree with the comments by @Roasbeef that the lack of global consensus should be added. 
also How was the list sorted? It is not alphanumerically

Reply:
> also How was the list sorted? It is not alphanumerically

sorry I put them in such an order because of usability and popularity. But yes this might be subjective. I have changed to alphanumerical order.